<ui version="4.0" >
 <class>ImportTrainingDataWorkingDlg</class>
 <widget class="QWidget" name="ImportTrainingDataWorkingDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="lbMain" >
     <property name="text" >
      <string>Der angegebene Ordner wird verarbeit.
Sie können den Fortschritt mit dem Fortschrittsbalken
überprüfen.

Bitte haben Sie einen Moment Geduld.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="pbMain" >
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="value" >
      <number>44449</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbStatus" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>68</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
